packages/js/data/src/onboarding/actions.js

```diff
@@ -260,6 +270,20 @@ export function getProductTypesError( error ) {
 	};
 }
 
+export function* keepCompletedTaskList( taskListId ) {
+	const updateOptionsParams = {
+		woocommerce_task_list_keep_completed: 'yes',
+	};
+	const response = yield controls.dispatch(
```

Comment:
Probably not something we can change in this PR, but another great case for our generic dismissal/modal API.